client/contracts/bindings/consensus/consensus.go

```diff
@@ -239,84 +217,76 @@ func (consensus *consensus) Deposits(addr common.Address) ([]*types.Deposit, err
 	return deposits, nil
 }
 
-func (consensus *consensus) IsGenesisValidator(address common.Address) (bool, error) {
-	return consensus.manager.IsGenesisValidator(&bind.CallOpts{}, address)
+func (css *Consensus) IsGenesisValidator(address common.Address) (bool, error) {
+	return css.manager.IsGenesisValidator(&bind.CallOpts{}, address)
 }
 
-func (consensus *consensus) IsValidator(address common.Address) (bool, error) {
-	return consensus.manager.IsValidator(&bind.CallOpts{}, address)
+func (css *Consensus) IsValidator(address common.Address) (bool, error) {
+	return css.manager.IsValidator(&bind.CallOpts{}, address)
 }
 
-func (consensus *consensus) MinimumDeposit() (*big.Int, error) {
-	return consensus.manager.GetMinimumDeposit(&bind.CallOpts{})
+func (css *Consensus) MinimumDeposit() (*big.Int, error) {
+	return css.manager.GetMinimumDeposit(&bind.CallOpts{})
 }
 
-func (consensus *consensus) GetValidatorCount() (*big.Int, error) {
-	return consensus.manager.GetValidatorCount(&bind.CallOpts{})
+func (css *Consensus) GetValidatorCount() (*big.Int, error) {
+	return css.manager.GetValidatorCount(&bind.CallOpts{})
 }
 
-func (consensus *consensus) MaxValidators() (*big.Int, error) {
-	return consensus.manager.MaxNumValidators(&bind.CallOpts{})
+func (css *Consensus) MaxValidators() (*big.Int, error) {
+	return css.manager.MaxNumValidators(&bind.CallOpts{})
 }
 
-func (consensus *consensus) Token() token.Token {
-	return consensus.mtoken
+func (css *Consensus) Token() token.Token {
+	return css.mtoken
 }
 
 //Minter interface implementation
 
-func (consensus *consensus) MintInit() error {
+func (css *Consensus) MintInit() error {
 	var err error
-	consensus.initMint.Do(func() {
-		if consensus.multiSigWallet == nil {
-			addr, ok := mapMiningTokenToAddr[consensus.chainID.Uint64()]
-			if !ok {
-				err = bindings.ErrNoAddress
-				return
-			}
-			consensus.mtokenAddr = addr
-
-			addr, ok = mapMultiSigWalletToAddr[consensus.chainID.Uint64()]
-			if !ok {
-				err = bindings.ErrNoAddress
-				return
-			}
+	css.initMint.Do(func() {
+		if css.multiSigWallet == nil {
+			addr := common.HexToAddress(bindings.MultiSigWalletAddr)
 
 			var multisig *ownership.MultiSigWallet
-			multisig, err = ownership.NewMultiSigWallet(addr, consensus.contractBackend)
+			multisig, err = ownership.NewMultiSigWallet(addr, css.contractBackend)
```

Comment:
why are we initilising the multi sig wallet in the consensus file?

Reply:
Not sure, this is not coming from my changes.